src/data/markdown/translated-guides/en/02 Using k6/14 Scenarios/04 Arrival Rate.md

```diff
@@ -3,15 +3,22 @@ title: 'Arrival rate'
 excerpt: 'In k6, we have implemented this open model with our two arrival rate executors: constant-arrival-rate and ramping-arrival-rate.'
 ---
 
+Different k6 executors have different ways of scheduling VUs.
+Some executors use the _closed model_, while the arrival rate executors use the _open model_.
+
+In short, in the closed model, VU iterations start only when the last iteration finishes.
+In the open model, on the other hand, VUs arrive independently of iteration completion.
+Different models suit different test aims.
+
 ## Closed Model
 
-> In a closed model, the execution time of each iteration dictates the actual
-> number of iterations executed in your test, as the next iteration won't be started
-> until the previous one is completed.
+In a closed model, the execution time of each iteration dictates the
+number of iterations executed in your test.
+The next iteration doesn't started until the previous one finishes.
 
-Prior to v0.27.0, k6 only supported a closed model for the simulation of new VU arrivals.
-In this closed model, a new VU iteration only starts when a VU's previous iteration has
-completed its execution. Thus, in a closed model, the start rate, or arrival rate, of
+Prior to v0.27.0, k6 supported only a closed model to simulate new VU arrivals.
+In this closed model, a new VU iteration starts only when a previous iteration finishes.
```

Comment:
This is now going to be repeated 3 times in the last 20 lines - here, and on lines 9 and 17.

Reply:
Good point, just deleted with https://github.com/grafana/k6-docs/commit/aa79e860842b3e11e4b4d2316f8096ccf8d08332.

By the way, thanks for the precise line-number reference. I discovered a new reason to favor one-sentence lines :trollface: .